Makefile

```
@@ -99,6 +99,11 @@ point-release:
 
 # Run via 'make VERSION_ADDITIONAL=rc2 release-custom' to specify a version string
 release-candidate:
+	@if [ "$(shell echo $(VERSION_ADDITIONAL) | head -c 1)" != "-" ]; \
+	then \
+	  echo "VERSION_ADDITIONAL must be specified and start with a \"-\" character. Eg. make VERSION_ADDITIONAL=-rc2 release-custom"; \
```

Super nitpick, but punctuation should be the same here vs the eg below

```suggestion
	  echo "VERSION_ADDITIONAL must be specified and start with a \"-\" character. Eg make VERSION_ADDITIONAL=-rc2 release-custom"; \
```